packages/dd-agent/packaging

```diff
@@ -2,51 +2,47 @@
 set -e -x
 
 # Grab the latest versions that are in the directory
-DD_AGENT_VERSION=$(ls -r dd-agent/dd-agent-*.tar.gz | sed 's/.*\/dd-agent-\(.*\)\.tar\.gz$/\1/' | head -1)
-BOSH_PACKAGES_DIR=${BOSH_PACKAGES_DIR:-/var/vcap/packages}
+DD_AGENT_VERSION=5.12.3
 JOB_NAME="dd-agent"
-LIBFFI_VERSION="3.2.1"
-
-# Integrate contents of all packages to this folder, so only this package needs
-# to be included in the job
-for pkg in $(ls --ignore=golang --ignore=dd-agent ${BOSH_PACKAGES_DIR}); do
-    echo "Integrating ${BOSH_PACKAGES_DIR}/${pkg}/* in ${BOSH_INSTALL_TARGET}"
-    cp -av ${BOSH_PACKAGES_DIR}/${pkg}/* ${BOSH_INSTALL_TARGET}
-done
+BOSH_PACKAGES_DIR=${BOSH_PACKAGES_DIR:-/var/vcap/packages}
 
+if [ -f /etc/debian_version -o "$DISTRIBUTION" == "Debian" -o "$DISTRIBUTION" == "Ubuntu" ]; then
```

Comment:
I don't think `DISTRIBUTION` is defined here. We can probably just keep the tests on the existence `/etc/debian-version` and `/etc/redhat-release` for now?